### PR TITLE
Add credentialless for iframe

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

File: docs/docs/home/get-started.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ rill start my-rill-project
   boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)"  /* Shadow effect */
 }}>
   <iframe
-    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
+    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg&&origin=https://docs.rilldata.com"
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:27]:
```suggestion
    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg&&origin=https://deploy-preview-5743--rill-developer.netlify.app"
```

File: docs/docs/home/get-started.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ rill start my-rill-project
   boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)"  /* Shadow effect */
 }}>
   <iframe
-    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
+    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg&&origin=https://deploy-preview-5743--rill-developer.netlify.app"
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:27]:
```suggestion
    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=E1Rkh7v2Arp8MCnS&&origin=https://deploy-preview-5743--rill-developer.netlify.app"
```

File: docs/docs/home/get-started.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ rill start my-rill-project
   boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)"  /* Shadow effect */
 }}>
   <iframe
-    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
+    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=E1Rkh7v2Arp8MCnS&&origin=https://deploy-preview-5743--rill-developer.netlify.app"
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:27]:
```suggestion
    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=E1Rkh7v2Arp8MCnS"
    credentialless
```

File: docs/docs/home/get-started.md [hunk 4]
```diff
@@ -24,7 +24,8 @@ rill start my-rill-project
   boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)"  /* Shadow effect */
 }}>
   <iframe
-    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
+    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=E1Rkh7v2Arp8MCnS"
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:27]:
```suggestion
    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
```

File: docs/docs/home/get-started.md [hunk 5]
```diff
@@ -25,6 +25,7 @@ rill start my-rill-project
 }}>
   <iframe
     src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
+    credentialless
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:28]:
```suggestion
  <iframe credentialless
    src="https://www.youtube.com/embed/GMsUyj_Bv3Q?autoplay=1&mute=1&rel=0&si=CMltjZI4S5oAAAtg"
```

File: netlify.toml [hunk 6]
```diff
@@ -7,7 +7,7 @@
       script-src 'self' https: 'unsafe-inline' 'unsafe-eval' *.usepylon.com *.posthog.com *.pusher.com; \
       style-src https: 'unsafe-inline'; \
       img-src https: data: blob:; \
-      frame-src 'self' https://www.youtube.com/ blob: data:; \
+      frame-src data:; \
```
Comment (himadrisingh) [inline netlify.toml:10]:
```suggestion
      frame-src 'self' https://www.youtube.com/ blob: data:; \
```

File: docs/docs/home/get-started.md [hunk 7]
```diff
@@ -23,7 +23,7 @@ rill start my-rill-project
   borderRadius: "15px",  /* Softer corners */
   boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)"  /* Shadow effect */
 }}>
-  <iframe
+  <iframe credentialless
```
Comment (himadrisingh) [inline docs/docs/home/get-started.md:26]:
```suggestion
  <iframe credentialless="true"
```

**Review by royendo — APPROVED** [review]
looks good :)

**Pull request merged**